test/support/api/clinicians.js

```diff
@@ -54,6 +54,7 @@ Cypress.Commands.add('routeCurrentClinician', (mutator = _.identity) => {
 Cypress.Commands.add('routeClinicians', (mutator = _.identity) => {
   cy.route({
     url: '/api/clinicians',
+    delay: 100,
```

Comment:
This should fix the flaky `clinicians list > find in list` test we were seeing.

This request was finishing before the test could check that the `Find in list...` input was disabled.

Reply:
I think the test just needs to be before the `wait` instead of after it.  I don't think we should introduce `delay`s unless it's css animation or completely outside of our control (form.io)

Reply:
That still results in the test failing about 1/2 the time 😞 

It's weird because this test was never flaky before (at least not that I noticed). So it must be something we changed recently with the `routeClinicians()` command.